axiom/nr_segment_traces.c

```diff
@@ -578,3 +587,65 @@ void nr_segment_traces_create_data(
 
   return;
 }
+
+/*
+ * Purpose : If available, add cloud attributes to segment.
+ *
+ * Params  : 1. segment to create and add agent attributes to
+ *           2. nr_segment_cloud_attrs_t* that contains the attributes
+ *
+ * Returns : void
+ *
+ */
+extern void nr_segment_traces_add_cloud_attributes(
```

Comment:
Why is this function defined in this file? It has nothing to do with segment traces.